deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -41,30 +46,44 @@ public abstract class OpenApiGeneratorCodeGenBase implements CodeGenProvider {
 
     private static final String DEFAULT_PACKAGE = "org.openapi.quarkus";
 
+    /**
+     * The input base directory from <pre>src/main<pre> directory. Ignored if INPUT_BASE_DIR is specified.
+     **/
     @Override
     public String inputDirectory() {
         return "openapi";
```

Comment:
why not using the new property here? What you want to change is the relative path, this is the right place to do so

Reply:
not really, because this path is relative to src/main, and I want to do it relative to the module